Cray compiler don't like having $LINKER set to a non-absolute path
author: Kenneth Hoste (HPC-UGent)
--- hpl-2.1/makes/Make.ptest.orig	2015-04-25 13:46:18.668624435 +0200
+++ hpl-2.1/makes/Make.ptest	2015-04-25 13:47:20.958060830 +0200
@@ -73,7 +73,7 @@
 	( $(CP) ../HPL.dat $(BINdir) )
 #
 dexe.grd: $(HPL_pteobj) $(HPLlib)
-	$(LINKER) $(LINKFLAGS) -o $(xhpl) $(HPL_pteobj) $(HPL_LIBS)
+	LINKER=`which ld` $(LINKER) $(LINKFLAGS) -o $(xhpl) $(HPL_pteobj) $(HPL_LIBS)
 	$(MAKE) $(BINdir)/HPL.dat
 	$(TOUCH) dexe.grd
 #
